docs/Getting Started/Integration Basics/index.md

```diff
@@ -1009,6 +1009,25 @@ This is only for specific OEM applications, therefore normal developers do not n
 
 Some OEMs choose to implement custom router services. Setting the `sdl_custom_router` metadata value to `true` means that the app is using something custom over the default router service that is included in the SDL Android library. Do not include this `meta-data` entry unless you know what you are doing.
 
+The final router service entry in the `AndroidManifest.xml` file should look like the following:
+```xml
+<service
+    android:name=".SdlRouterService"
+    android:enabled="true"
```

Comment:
Does this need to be added to Hello Sdl Android?

Reply:
We could but it is not very necessary as the default value for "enabled" is true by default: https://developer.android.com/guide/topics/manifest/service-element